modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,199 @@
+
+
+== Enable Iceberg integration
+
+To use the Iceberg integration for a topic, you must set Redpanda cluster configuration `enable_datalake` to true, and also explicitly set the topic configuration to `true`.
+
+. Enable the Iceberg integration for the cluster by setting the cluster configuration `enable_datalake` to `true`. 
++
+[,bash]
+----
+rpk cluster config set enable_datalake true 
+----
++
+[,bash,role=no-copy]
+----
+Successfully updated configuration. New configuration version is 2.
+----
+
+. Create a new topic.
++
+[,bash,]
+----
+rpk topic create <new-topic-name> --partitions 1 --replicas 1
+----
++
+[,bash,role=no-copy]
+----
+TOPIC            STATUS
+new-topic-name   OK
+----
+
+. Enable the integration for the topic. 
++
+[,bash]
+----
+rpk topic alter-config <new-topic-name> --set iceberg_topic=true 
+----
++
+[,bash,role=no-copy]
+----
+TOPIC            STATUS
+new-topic-name   OK
+----
+
+. Optional: Register a schema for the topic. Redpanda uses xref:manage:schema-reg/schema-id-validation.adoc#set-subject-name-strategy-per-topic[`TopicNameStrategy`] (`<topic-name>-value` or `<topic-name>-key`) to find the matching schema based on the subject name. Redpanda derives the Iceberg table format from the schema and writes data files to object storage in this format. Protobuf and Avro schemas are supported.
++
+If there is no registered schema for the topic, Redpanda by default uses a simple schema consisting of the record’s key, value, and the original event timestamp. If you don't have a need for a strict schema and can use the data in a more semi-structured format, consider the schemaless approach.
++
+[,bash]
+----
+rpk registry schema create <subject-name> --schema </path-to-schema> --type <format>
+----
++
+[,bash,role=no-copy]
+----
+SUBJECT                VERSION   ID   TYPE
+new-topic-name-value   1         1    PROTOBUF
+----
+
+As you produce records to the topic, the data also becomes available to consume from object storage by Iceberg-compatible clients. 
+
+== Consume data in Iceberg tables
+
+You can use the same analytical tools to access table data just like you would in a relational database or data lake.
+
+For example, suppose you are producing the same stream of events to a topic `test_topic`, and another topic `test_topic_schemaless`. The topics have Tiered Storage configured to an AWS S3 bucket. A record looks like the following:
+
+[,bash,role=no-copy]
+----
+
+----
+
+When you point your Iceberg-compatible tool or framework to the object storage location of the Iceberg tables, how you consume the data depends on whether you've registered a schema on which to base the table format. In either approach, you do not need to rely on complex ETL jobs or pipelines to consume real-time data from Redpanda.
+
+=== Structured approach with schema
+
+If you register the following schema for `test_topic` under the `test_topic-value` subject:
```

Comment:
This feels like an incomplete sentence. I can see the example relates to the section above but not really sure how.